aws-flow/Gemfile.lock

```diff
@@ -0,0 +1,88 @@
+GIT
```

Comment:
I don't believe the lock file is part of this repo. Is there a reason you added it now?